### PR TITLE
New first login implementation 

### PR DESCRIPTION
**Pull request by dimuthud**

### CODE REVIEW

File: jaggeryapps/admin/templates/header.jag [hunk 1]
```diff
@@ -181,14 +181,18 @@ var billing = billing || (function () {
...
+                 if (!billingResult.data.error) {
+                     if (log.isDebugEnabled()) {
+                           log.debug("Trial account activated for tenant: " + tenantDomain);
+                     }
 
                  isExpired = false;
                  isTrial = true;
                  accountContent = "Trial – " + (trialPeriod)  + " days to upgrade";
+
+                 }else {
```
Comment (danushkaf) [inline jaggeryapps/admin/templates/header.jag]: Fix formatting issues.

File: jaggeryapps/admin/templates/header.jag [hunk 2]
```diff
@@ -181,14 +181,18 @@ var billing = billing || (function () {
...
-
-                 if (log.isDebugEnabled()) {
-                      log.debug("Trial account activated for tenant: " + tenantDomain);
-                 }
+                 if (!billingResult.data.error) {
+                     if (log.isDebugEnabled()) {
+                           log.debug("Trial account activated for tenant: " + tenantDomain);
+                     }
 
                  isExpired = false;
```
Comment (danushkaf) [inline jaggeryapps/admin/templates/header.jag]: Fix formatting

Comment (dimuthud) [inline jaggeryapps/admin/templates/header.jag]: Fixed.

File: jaggeryapps/admin/templates/header.jag (hunk 1 above)
Comment (dimuthud) [inline jaggeryapps/admin/templates/header.jag]: Fixed.

**Review by danushkaf — APPROVED** [review]

**Pull request merged**